fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1304,5 +1304,18 @@
     @Deprecated
     @ConfField(mutable = true)
     public static boolean enable_vectorized_file_load = true;
-}
 
+    /**
+     * Whether to collect routine load process metrics.
+     * Be careful to turn this on, because this will call kafka api to get the partition's latest offset.
+     */
+    @ConfField(mutable = true)
+    public static boolean enable_routine_load_lag_metrics = false;
```

Comment:
when to not use this metric?

Reply:
1. If the user has his own monitoring of the Kafka consumer
2. There are many routine load jobs. Access to the metrics interface can be slow, because each job has to access kafka

Reply:
Can we get Kafka's all partition messages in one batch instead of one by one?

Reply:
Sure, I'll optimize it

Reply:
done